ambry-rest/src/main/java/com/github/ambry/rest/PublicAccessLogger.java

```diff
@@ -22,11 +22,11 @@
  */
 public class PublicAccessLogger {
 
-  private Logger publicAccessLogger = LoggerFactory.getLogger("PublicAccessLogger");
+  private static final Logger publicAccessLogger = LoggerFactory.getLogger("PublicAccessLogger");
 
   private final String[] requestHeaders;
   private final String[] responseHeaders;
-  private final Logger logger = LoggerFactory.getLogger(getClass());
+  private static final Logger logger = LoggerFactory.getLogger(PublicAccessLogger.class);
```

Comment:
We can probably remove this second logger, it's only used in one trace message.

Reply:
Good idea, that was not a very useful log message anyways.